packages/lib-sourcify/src/lib/CheckedContract.ts

```diff
@@ -423,6 +423,14 @@ export class CheckedContract {
       this.solcJsonInput,
       forceEmscripten
     );
+    if (this.compilerOutput === undefined) {
```

Comment:
Here sometimes the lambda compiler returns undefined and creates weird `cannot get ".compiler" from undefiend` errors.